build.fsx

```diff
@@ -418,14 +406,28 @@ let githubRelease() =
     | Some user, Some token ->
         async {
             try
-                let ghreleases: GhRealeases = JsInterop.importAll "ghreleases"
                 let! version, notes = Publish.loadReleaseVersionAndNotes "src/Fable.Cli"
-                run <| sprintf "git commit -am \"Release %s\" && git push" version
-                let! res = ghreleases.create(user, token, "fable-compiler", "Fable", version, String.concat "\n" notes) |> Async.AwaitPromise
+                // TODO: escape single quotes
+                let notes = notes |> Array.map (sprintf "'%s'") |> String.concat ","
+                run $"git commit -am \"Release {version}\" && git push"
+                runSilent $"""
+node --eval "require('ghreleases').create({{
+    user: '{user}',
+    token: '{token}',
+}}, 'fable-compiler', 'Fable', {{
+    tag_name: '{version}',
+    name: '{version}',
+    body: [{notes}].join('\n'),
+}}, (err, res) => {{
+    if (err != null) {{
+        console.error(err)
+    }}
+}})"
+"""
```

Comment:
Did I say that I love string interpolation already?

Reply:
We got some serious meta-programming right here! 😄